lib/src/widgets/nes_icon.dart

```diff
@@ -142,6 +142,20 @@ class NesIcons {
       '8,8;2,-1;4,0;3,-1;1,0;4,-1;1,0;1,-1;1,0;6,-1;1,0;2,-1;1,0;4,-1;1,0;1,-1;1,0;1,-1;1,0;3,-1;2,0;1,-1;1,0;1,-1;1,0;2,-1;1,0;2,-1;1,0;3,-1;1,0;4,-1;3,0;2,-1',
     ),
   );
+
+  /// An unpressed button icon.
+  late final unpressedButton = NesIconData(
```

Comment:
Just a quick question here, for my understanding... 
Why do you set this as a `late` if you are already initializing it here?

Reply:
by adding the `late` we can use an awesome feature from dart that is the lazy initialization.

So, my code indeed is initializing it, but when this is in runtime, dart will only really initialize the variable when it is accessed for the first time.

So by doing so, we will only allocate the icons in memory that the user is using.

Reply:
Got it! Thanks for explaining! 🤗